packages/@adobe/spectrum-css-temp/components/slider/index.css

```diff
@@ -323,6 +324,10 @@ governing permissions and limitations under the License.
   cursor: default;
   font-feature-settings: "tnum";
   text-align: end;
+
+  &:only-child {
```

Comment:
What's the `:only-child` for?

Reply:
When it is a side label, the value and label are in separate label containers... the way I added the start align to the value when side label is that value is the only child of a label container.

Reply:
Oh I assumed there was a class for the side label state but I guess not... 😞 

Maybe just add a comment to explain what you just said then.

Reply:
Maybe we should use the `spectrum-Slider--positionSide` class now?